mobile/build.gradle

```diff
@@ -56,11 +56,14 @@ android {
     }
     buildTypes {
         release {
-            minifyEnabled false
+            minifyEnabled true
+            shrinkResources true
             proguardFiles getDefaultProguardFile("proguard-android.txt"), "proguard-rules.pro"
         }
         debug {
-            minifyEnabled false
+            minifyEnabled true
+            shrinkResources true
+            proguardFiles getDefaultProguardFile("proguard-android.txt"), "proguard-rules.pro"
```

Comment:
Why minify the debug build? We don't ship it to user's, and from my experience it'll make debugging more complicated (e.g. due to optimized out variables and the like).

Reply:
I wanted to minify the debug build, so issues with removing too much resources or classes can be seen when debugging the app, but I can revert it.